File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& arr) {
        int j=1;
        sort(arr.begin(),arr.end());
        for(int i=1;i<arr.size();i++ ){
            if(arr[i]>arr[i-1]){
                arr[j]=arr[i];
                j++;
            }
        }
        return j;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """


        j=1
        n=len(nums)
        for i in range(1,n):
            if nums[i]!=nums[i-1]:
                nums[j]=nums[i]
                j=j+1

        return j            
            

        